girder/api/v1/assetstore.py

```diff
@@ -39,6 +40,7 @@ def __init__(self):
         self.route('PUT', (':id',), self.updateAssetstore)
         self.route('DELETE', (':id',), self.deleteAssetstore)
         self.route('GET', (':id', 'files'), self.getAssetstoreFiles)
+        self.route('GET', (':id', 'spaceusage',), self.getSpaceUsageStatistics)
```

Comment:
I see. In that case, did you also think we should push space usage one level down in the JSON result?